app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,5 @@
+class CategoriesController < ApplicationRecord
```

Comment:
I think file should be removed. This confused concerns of a controller and model.